server/src/addrs.rs

```diff
@@ -57,7 +58,7 @@ impl BackendAddr {
         self.0.to_url().to_string()
     }
 
-    #[args(path = "\"\"")]
+    #[pyo3(signature = (path = "\"\""))]
```

Comment:
```suggestion
    #[pyo3(signature = (path = ""))]
```

The default value is `""` (empty string) it's different from `'""'` (string containing `""`)

Reply:
Missed, done